src/constructs/iam/policies/ses.ts

```diff
@@ -0,0 +1,22 @@
+import { Effect, PolicyStatement } from "@aws-cdk/aws-iam";
+import type { GuStack } from "../../core";
+import { GuGuardianEmailSenderParameter } from "../../core";
+import type { GuPolicyProps } from "./base-policy";
+import { GuPolicy } from "./base-policy";
+
+export class GuSESSenderPolicy extends GuPolicy {
+  constructor(scope: GuStack, id: string = "GuSESSenderPolicy", props?: GuPolicyProps) {
+    super(scope, id, { ...props });
+
+    const emailSenderParam = new GuGuardianEmailSenderParameter(scope);
+
+    this.addStatements(
+      // see https://docs.aws.amazon.com/ses/latest/DeveloperGuide/sending-authorization-policies.html
+      new PolicyStatement({
+        effect: Effect.ALLOW,
+        actions: ["ses:SendEmail"],
```

Comment:
I remember there was one example from the datalake where they had both this and SendRawEmail. I was reading up on the difference in the [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html) and SendRawEmail is what you need to use to send an email with an attachment, for example. Do you think it's overkill to include it here as well?

Reply:
Fair enough!